README.md

```diff
@@ -55,7 +60,9 @@ Ordinarily, scriptworker would get the task definition from TaskCluster, and wri
 The important entries to edit are the:
  * `apks`: point to the file(s) to publish to Google Play
  * `dependencies`: need to match the `taskId`s of the URLs unless you modify the `valid_artifact_*` config items as specified above
- * `scopes`: the first and only scope, `project:releng:googleplay:*`, tells which product in Google Play store should be updated (either [aurora](https://play.google.com/store/apps/details?id=org.mozilla.fennec_aurora), [beta](https://play.google.com/store/apps/details?id=org.mozilla.firefox_beta), or [release](https://play.google.com/store/apps/details?id=org.mozilla.firefox))
+ * `scopes`: 2 scopes are needed:
```

Comment:
You can continue using 2 scopes, or you can use one, and automatically pick the cert to use based on the google play product.

I would lean towards a single scope of release/beta/nightly (or even release/beta/aurora, which is more accurate), and have the `script_config.json` specify that for beta and aurora, we use the nightly key alias, and for release, we use the release key alias.  We can keep it as is, but we might want to add a check to make sure we're not using the wrong key for the wrong product.

Finally, we'd need to make sure the scopes we want to restrict are in the dict [here](https://github.com/mozilla-releng/scriptworker/blob/master/scriptworker/constants.py#L177-L188).  If we use release/beta/aurora as the scopes, we'll want to add `beta` and `aurora` keys [here](https://github.com/mozilla-releng/scriptworker/blob/master/scriptworker/constants.py#L196-L203), to limit which trees can use each one... probably just mozilla-aurora for aurora, and mozilla-beta for beta?  Let me know if you want help with this part.   Or if you have questions about any of this.